sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/ConstraintSuite.scala

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.connector.catalog
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.connector.catalog.constraints.Constraint
+import org.apache.spark.sql.connector.catalog.constraints.Constraint.ValidationStatus
+import org.apache.spark.sql.connector.expressions.{Expression, FieldReference, LiteralValue, NamedReference}
+import org.apache.spark.sql.connector.expressions.filter.Predicate
+import org.apache.spark.sql.types.IntegerType
+
+class ConstraintSuite extends SparkFunSuite {
+
+  test("CHECK constraint toDDL") {
+    val con1 = Constraint.check("con1")
+      .sql("id > 10")
+      .enforced(true)
+      .validationStatus(ValidationStatus.VALID)
+      .rely(true)
+      .build()
+    assert(con1.toDDL == "CONSTRAINT con1 CHECK id > 10 ENFORCED VALID RELY")
```

Comment:
There should be `()` wrapped around the check condition `id > 10`

Reply:
Good catch, fixed.